model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/authorization/ResourceServerAdapter.java

```diff
@@ -32,11 +32,14 @@
 public class ResourceServerAdapter implements ResourceServer, CachedModel<ResourceServer> {
     protected CachedResourceServer cached;
     protected StoreFactoryCacheSession cacheSession;
+    private RealmModel realm;
     protected ResourceServer updated;
 
-    public ResourceServerAdapter(CachedResourceServer cached, StoreFactoryCacheSession cacheSession) {
+    public ResourceServerAdapter(CachedResourceServer cached, StoreFactoryCacheSession cacheSession,
+            RealmModel realm) {
```

Comment:
In map storage, when using `RealmModel` as parameter, it's always as first parameter, So in sake of consistency:
```suggestion
    public ResourceServerAdapter(RealmModel realm, CachedResourceServer cached,
            StoreFactoryCacheSession cacheSession) {
```

Reply:
I would prefer the same